src/ai/backend/storage/spawn-ceph-clusters/README.md

```diff
@@ -0,0 +1,94 @@
+# Readme
```

Comment:
The scripts inside `spawn-ceph-clusters` seems very useful to those who want to bootstrap ceph cluster, but I don't think it's not that related to the source code itself. Perhaps you can move this directory onto `scripts/stroage-proxy` directory of the repository.